### PR TITLE
podvm: agent policy support

### PR DESCRIPTION
**Pull request by snir911**
[KATA-3076](https://issues.redhat.com//browse/KATA-3076)

### CODE REVIEW

File: config/peerpods/podvm/lib.sh [hunk 1]
```diff
@@ -195,6 +195,12 @@ function prepare_source_code() {
         sed -i '/exit 0/ifips-mode-setup --enable' "${podvm_dir}"/qcow2/misc-settings.sh ||
             error_exit "Failed to enable fips mode"
     fi
+
+    if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
+        sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/; s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
+            "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
```
Comment (gkurz) [inline config/peerpods/podvm/lib.sh:201]: Assuming that `default-policy.rego` comes from https://github.com/confidential-containers/cloud-api-adaptor/tree/main/src/cloud-api-adaptor/podvm/files/etc/kata-opa, this looks good to me.

**Review by gkurz — APPROVED** [review]
/lgtm

Thanks Snir !

Is there a jira to link to BTW ?

**Comment by snir911** [comment]
I removed the option for custom policy ATM as it messes something and i do not want to block the more essential part of this PR

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**